python/snewpy/scripts/snewpy_to_marley.py

```diff
@@ -0,0 +1,95 @@
+
+import logging
+import os
+import re
+import tarfile
+import io
+import json
+
+import numpy as np
+import pandas as pd
+from astropy import units as u
+from warnings import warn
+
+from snewpy.models.ccsn import Bollig_2016
+from snewpy.flavor_transformation import AdiabaticMSW
+from snewpy.neutrino import MixingParameters
+from snewpy.flavor import ThreeFlavor
+
+def marley_name(flavor):
+    marley_name = "v"
+    if 'E' in flavor.name:
+        marley_name += "e"
+    elif 'MU' in flavor.name:
+        marley_name += "u"
+    else:
+        marley_name += "t"
+    if 'BAR' in flavor.name:
+        marley_name += "bar"
+
+    return marley_name  
+    
+def save_as_marley(fluence_object, output_filename, marley_additional_inputs=None):
+    """Save the contents of a fluence object in MARLEY format.
+
+       For each time bin 6 Marley config files are genreated and placed into a tarfile.
+
+       Parameters
+       ----------
+       fluence_object : flux container
+           Input file containing neutrino flux information from supernova model.
+       marley_additional_inputs : dict
```

Comment:
It might be nice to rename this to `additional_inputs`, to be a bit less unwieldy?